fs_attachment_s3_migration/tests/test_attachment_migration.py

```
@@ -0,0 +1,311 @@
+"""Tests for S3 attachment migration functionality."""
+
+from unittest.mock import MagicMock, patch
+
+from odoo.tests.common import TransactionCase
+
+
+class TestS3MigrationDomain(TransactionCase):
+    """Test suite for migration domain building."""
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.14.10)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)